homeassistant/components/plex/services.py

```diff
@@ -19,12 +27,20 @@ async def async_setup_services(hass):
     async def async_refresh_library_service(service_call):
         await hass.async_add_executor_job(refresh_library, hass, service_call)
 
+    async def async_scan_clients_service(_):
+        _LOGGER.info("Scanning for new Plex clients")
```

Comment:
We rarely log with info level in integrations. Standard is to use debug level. Info is ok if a device comes back from unavailable to become available.